algosdk/logic.py

```diff
@@ -23,7 +23,11 @@ def check_program(program, args=None):
     Raises:
         InvalidProgram: on error
     """
+    ok, _, _ = read_program(program, args)
+    return ok
 
+
+def read_program(program, args=None):
```

Comment:
I like this and am pulling it into my open PRs (see team slack discussion). Does `read_program` ever return `False` for the first return value `ok`? I don't see it if so

Reply:
I see the `check program` it's based on does the same thing, "returns true, else throws". Ok I get the context now :+1: